src/bokeh/models/annotations/labels.py

```diff
@@ -161,6 +186,28 @@ def __init__(self, *args, **kwargs) -> None:
     distance in |screen units| from a given data position.
     """)
 
+    angle = Angle(default=0, help="""
+    The angle to rotate the text, as measured from the horizontal.
+    """)
+
+    angle_units = Enum(AngleUnits, default="rad", help="""
+    Acceptable values for units are ``"rad"`` and ``"deg"``.
+    """)
+
+    direction = Enum(Direction, default="anticlock", help="""
+    The direction of rotation of the label.
```

Comment:
```suggestion
    The direction for the angle of the label.
```
At least for me "rotation" more strongly invokes the act of changing the angle, and didn't immediately seem related to the interpreation of `angle` property